### PR TITLE
all: bump to Go-1.19

### PR DESCRIPTION
**Pull request by sbinet**
Please take a look.

### CODE REVIEW

**Comment by sbinet** [comment]
@kortschak, do you understand why Go-1.19's test fails with `Process completed with exit code 1.` while there are no error being reported by `go test ./...` ?

(also: could you please remove the expected flag for `1.17.x` ? thanks)

**Comment by kortschak** [comment]
I have no idea why that is failing. It passes locally.

**Comment by kortschak** [comment]
Maybe try running the tests if `-v` to see if that gives a hint. Otherwise, bisecting the packages to see which package is causing the problem might help.

**Comment by sbinet** [comment]
aha! that was actually `test -z "$(gofmt -d .)"` (GitHub's CI UI was greatly misleading...)

**Comment by sbinet** [comment]
PTAL

**Comment by sbinet** [comment]
I guess it doesn't handle well shell builtin functions ?
(anyways, there was also a bug in `.ci/check-formatting.sh` which should have made the initial error easier to spot)

File: plotter/boxplot.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ type BoxPlot struct {
 // NewBoxPlot returns a new BoxPlot that represents
 // the distribution of the given values.  The style of
 // the box plot is that used for Tukey's schematic
-// plots is ``Exploratory Data Analysis.''
+// plots is “Exploratory Data Analysis.”
```
Comment (kortschak) [inline plotter/boxplot.go:90]:
```suggestion
// plots in “Exploratory Data Analysis.”
```

**Review by kortschak — COMMENTED** [review]
There are cases where we are using \<pre\> blocks to do lists. I think those can be done with the godoc mark up now.

**Comment by sbinet** [comment]
PTAL

**Review by kortschak — APPROVED** [review]

**Comment by sbinet** [comment]
(force-push is only to clean up git history, no code change)

**Pull request merged**